### PR TITLE
Fix broken links to dresden-elektronik.de + minor edits

### PR DESCRIPTION
**Pull request by manup**
**Description:**

Fix broken links related to the dresden-elektronik.de website relaunch. Further streamline capitalization of product names.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/deconz.markdown [hunk 1]
```diff
@@ -43,7 +43,7 @@ See [deCONZ wiki](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/
 
 Home Assistant will automatically discover deCONZ presence on your network, if `discovery:` is present in your `configuration.yaml` file.
 
-If you don't have the API key, you can generate an API key for deCONZ by using the one-click functionality similar to Philips Hue. Go to **Settings** -> **Gateway** -> **Advanced** -> **Authenticate app** in deCONZ and then use the deCONZ configurator in Home Assistant frontend to create an API key. When you're done setting up deCONZ it will be stored as a config entry.
+If you don't have the API key, you can generate an API key for deCONZ by using the one-click functionality similar to Philips Hue. Go to **Settings** &rarr; **Gateway** &rarr; **Advanced** &rarr; **Authenticate app** in the Phoscon App and then use the deCONZ configurator in Home Assistant frontend to create an API key. When you're done setting up deCONZ it will be stored as a config entry.
```
Comment (frenck) [inline source/_integrations/deconz.markdown]: Either use the previous arrows, or insert the UTF8 ones.
This is harder to understand for people who read the Markdown.

Preferably the Markdown should be just as easy to read as the website.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @manup!

Left a small comment, if you could take a look? 👍

File: source/_integrations/deconz.markdown (hunk 1 above)
Comment (manup) [reply source/_integrations/deconz.markdown]: Agree, it was a bit too cryptic, I've changed them to UTF-8 now.

**Review by frenck — APPROVED** [review]
Thanks, @manup! 👍

**Pull request merged**